lib/pages/home/pins.dart

```diff
@@ -243,3 +226,66 @@ class _PinItemWidget extends StatelessWidget {
     );
   }
 }
+
+class ReadMoreWrapper extends StatefulWidget {
+  const ReadMoreWrapper(this.content, {Key? key}) : super(key: key);
+  final String content;
```

Comment:
Add an empty line before this.

Reply:
Two lines before class or an empty line before constructor ?
This is format by dart.